yarn.lock

```diff
@@ -3,83 +3,83 @@
 
 
 "@babel/code-frame@^7.0.0", "@babel/code-frame@^7.14.5":
-  version "7.15.8"
```

Comment:
Do you have any idea why every line in this file was changed? With SvelteKit being so new, untested updates have been caused bugs in the past.

Reply:
I switched to a previous commit before updating svelte kit and all that and redid the function. Please check to see if they all work out.